leme-onboarding/src/components/Validation.tsx

```diff
@@ -0,0 +1,87 @@
+
+
+export function validationEmail(email: string) {
+    /**
+     * 0 - Validated Email
+     * 1 - Empty field
+     * 2 Email not validated
+    */
```

Comment:
Talvez se você criar constantes para cada um desses números no início do arquivo, não precisaria dos comentários:

```ts
const VALIDATED_EMAIL = 0;
const EMPTY_FIELD = 1;
const EMAIL_NOT_VALIDATED = 2;
```

Mesma coisa para a senha.

Reply:
Resolvido!